packages/api-explorer/src/scenes/MethodScene/MethodScene.tsx

```diff
@@ -108,8 +108,14 @@ export const MethodScene: FC<DocMethodProps> = ({ api }) => {
         <Aside width="50rem">
           <ComponentsProvider
             globalStyle={false}
+            loadGoogleFonts
```

Comment:
This should not `loadGoogleFonts` - it only needs to be done once per application.